src/main/java/com/commercetools/project/sync/model/LastSyncCustomObject.java

```diff
@@ -77,4 +79,24 @@ public void setApplicationVersion(@Nonnull final String applicationVersion) {
   public void setLastSyncDurationInMillis(final long lastSyncDurationInMillis) {
     this.lastSyncDurationInMillis = lastSyncDurationInMillis;
   }
+
+  // TODO: Also include statistics in equals comparison after
+  // https://github.com/commercetools/commercetools-sync-java/issues/376 is resolved
```

Comment:
why application version is not important?

Reply:
thanks. I missed it: 
65c9332